app/src/androidTest/java/com/github/houseorganizer/houseorganizer/FirebaseTestsHelper.java

```diff
@@ -138,6 +142,64 @@ protected static void createTestTaskList() throws ExecutionException, Interrupte
+    /**
+     * This method will create events for testing
+     */
+    protected static void createTestEvents() throws ExecutionException, InterruptedException {
+        FirebaseFirestore db = FirebaseFirestore.getInstance();
+
+        Map<String, Object> toDeleteData = new HashMap<>();
+        toDeleteData.put("title", "title");
+        toDeleteData.put("description", "desc");
+        toDeleteData.put("start", LocalDateTime.of(2022, 10, 10, 11, 10).toEpochSecond(ZoneOffset.UTC));
+        toDeleteData.put("duration", 10);
+        toDeleteData.put("household", db.collection("households").document(TEST_HOUSEHOLD_NAMES[0]));
+        Map<String, Object> hasAttachmentData = new HashMap<>();
+        hasAttachmentData.put("title", "title");
+        hasAttachmentData.put("description", "desc");
+        hasAttachmentData.put("start", LocalDateTime.of(2022, 10, 10, 8, 10).toEpochSecond(ZoneOffset.UTC));
+        hasAttachmentData.put("duration", 10);
+        hasAttachmentData.put("household", db.collection("households").document(TEST_HOUSEHOLD_NAMES[0]));
+        Map<String, Object> noAttachmentData = new HashMap<>();
+        noAttachmentData.put("title", "title");
+        noAttachmentData.put("description", "desc");
+        noAttachmentData.put("start", LocalDateTime.of(2022, 10, 10, 9, 10).toEpochSecond(ZoneOffset.UTC));
+        noAttachmentData.put("duration", 10);
+        noAttachmentData.put("household", db.collection("households").document(TEST_HOUSEHOLD_NAMES[0]));
+        Map<String, Object> toDeleteAttachmentData = new HashMap<>();
+        toDeleteAttachmentData.put("title", "title");
+        toDeleteAttachmentData.put("description", "desc");
+        toDeleteAttachmentData.put("start", LocalDateTime.of(2022, 10, 10, 10, 10).toEpochSecond(ZoneOffset.UTC));
+        toDeleteAttachmentData.put("duration", 10);
+        toDeleteAttachmentData.put("household", db.collection("households").document(TEST_HOUSEHOLD_NAMES[0]));
+
+        Task<Void> task1 = db.collection("events").document("to_delete").set(toDeleteData);
+        Task<Void> task2 = db.collection("events").document("has_attachment").set(hasAttachmentData);
+        Task<Void> task3 = db.collection("events").document("no_attachment").set(noAttachmentData);
+        Task<Void> task4 = db.collection("events").document("to_delete_attachment").set(toDeleteAttachmentData);
+        Tasks.await(task1);
+        Tasks.await(task2);
+        Tasks.await(task3);
+        Tasks.await(task4);
+    }
+
+    /**
+     *  This method creates attachments linked to events for testing
+     */
+    protected static void createAttachments() throws ExecutionException, InterruptedException {
+        FirebaseStorage storage = FirebaseStorage.getInstance();
+
+        // For now a hardcoded bytestream instead of an image
+        // it will still create the popup just it wont display anything
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        baos.write(10000000);
```

Comment:
Maybe a smaller number could make the tests run faster ? 
But it is pretty unimportant

Reply:
That's true ill just write 1 for now